src/test/e2e-emulated/masterchain.spec.ts

```diff
@@ -122,7 +122,9 @@ describe("masterchain", () => {
         await system.run();
         const addr = new Address(-1, Buffer.alloc(32, 0));
         void expect(
-            contract.getParseAddress(beginCell().storeAddress(addr).endCell()),
+            contract.getParseAddress(
```

Comment:
Hmmm, looks like with this change the amount of boilerplate for tests has increased a bit. Can we do something about it? I'd expect that with more precise types it should become easier for the end-user to write tests

Reply:
> I'd expect that with more precise types it should become easier for the end-user to write tests

In general - yes. And not only tests but also custom code that work with contracts.
But in some cases (such as the one from this review comment) it'll require *more* actions, like transform `Cell` into a `Slice`.

I don't think we can do anything about such cases.

Reply:
ok, fair enough